features/netsocket/LoRaRadio.h

```diff
@@ -0,0 +1,402 @@
+/**
```

Comment:
This file should be alongside other lora code, not netsocket.

Reply:
This is a pure virtual class just like LoRaWANBase class. It can be used even if the feature lorawan is not available. If we move it to lorawan/ directory it would lose its purpose.